### PR TITLE
Streaming API Integration

### PR DESCRIPTION
**Pull request by andycreeth**

### CODE REVIEW

File: app/services/platforms/index.ts [hunk 1]
```diff
@@ -255,9 +258,12 @@ export function getPlatformService(platform: TPlatform): IPlatformService {
     facebook: FacebookService.instance,
     tiktok: TiktokService.instance,
     trovo: TrovoService.instance,
+    twitter: TwitterPlatformService.instance,
   }[platform];
 }
 
+window['getPlatformService'] = getPlatformService;
```
Comment (xoredg) [inline app/services/platforms/index.ts:265]: Might be a debugging left-over, but don't mind me if intended.

File: media/images/platforms/twitter-logo-black.png
Comment (xoredg) [inline media/images/platforms/twitter-logo-black.png:1]: Looks like these aren't used.

**Review by xoredg — APPROVED** [review]
This looks great! A couple of very small comments, nothing major.

File: app/services/platforms/index.ts (hunk 1 above)
Comment (andycreeth) [reply app/services/platforms/index.ts:266]: yup debugging - ty!

File: media/images/platforms/twitter-logo-black.png
Comment (andycreeth) [reply media/images/platforms/twitter-logo-black.png:1]: yeah - we switched to CDN links.  I'm going to keep them in here so we can switch back to repo links once we've shipped 1.15.0

**Review by holiber — APPROVED** [review]

**Pull request merged**